pybv/io.py

```diff
@@ -255,8 +256,10 @@ def _scale_data_to_unit(data, units):
         scale = SUPPORTED_UNITS.get(unit, None)
         if scale is None:
             msg = (f'Encountered unsupported unit: {unit}\n'
-                   f'Use one of the following: {set(SUPPORTED_UNITS.keys())}')
-            raise ValueError(msg)
+                   f'We are not scaling channel {idx}. If this is a mistake, '
+                   f'use one of the following: {set(SUPPORTED_UNITS.keys())}')
```

Comment:
can we do something like (you'd have to adjust for correct line length):

```python
f'use one of the following: {', '.join(sorted(SUPPORTED_UNITS.keys()))}')
```
you don't need `set()` as dict keys need to be unique anyway.